corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py

```diff
@@ -1977,8 +1977,8 @@ def create_case_graph(num):
         p2, o2, e2 = create_case_graph(2)
 
         # Alice and Bob sync
-        alice = self.get_device()
-        bob = self.get_device(user=self.other_user)
+        alice = self.get_device(sync=True)
```

Comment:
Why isn't this just

```python
alice = self.get_device()
alice.sync()
```

I guess it's more lines. But it is more explicit about what's happening.